esp/services/ws_workunits/ws_workunitsHelpers.hpp

```diff
@@ -365,6 +365,7 @@ struct CWsWuZAPInfoReq
     StringBuffer emailFrom, emailTo, emailServer, emailSubject, emailBody;
 
     bool sendEmail, attachZAPReportToEmail;
+    bool includeRelatedLogs = true, includePerComponentLogs = false;
```

Comment:
For backward compatibility I think this needs to default to true.

Reply:
@rpastrana please comment (referring https://hpccsystems.atlassian.net/browse/HPCC-31441?focusedCommentId=55731).

Reply:
the reason for this change is to avoid unnecessary per-pod queries unless the user explicitly requests them.

Reply:
@wangkx was your feature to add the per-pod logs documented anywhere? if so, we will need to update the documentation as well.